helpers.py

```diff
@@ -100,13 +133,13 @@ def update_modified(subject, modified=datetime.datetime.now()):
     update(query)
 
     query = " INSERT DATA {"
-    query += "   GRAPH <%s> {" % graph
+    query += "   GRAPH <%s> {" % MU_APPLICATION_GRAPH
     query += "     <%s> <%s> %s ." % (subject, DC.Modified, sparql_escape(modified))
     query += "   }"
     query += " }"
     update(query)
 
-
+# TODO: Not sure what this function is for, since strings are supposed to be escaped.
```

Comment:
Hey Michael, I missed this during our original review. From the look of it it seems to be a method to prevent SPARQL-injection. Since we do not support this in other templates, can you delete this method entirely?

Reply:
Method has been deleted.